ocaml/xapi/xapi_cluster_host.mli

```diff
@@ -69,3 +69,8 @@ val disable_clustering : __context:Context.t -> unit
 (** [disable_clustering ~__context] is a wrapper for Xapi_cluster_host.disable
     which finds the local cluster_host [self], calls [disable ~__context self]
     and logs its actions. *)
+
+val resync_host : __context:Context.t -> host:API.ref_host -> unit
+(** [resync_host ~__context ~host] calls [create_if_necessary ~__context ~host],
```

Comment:
Add a comment here saying that this is a no-op if no Cluster objects exists, i.e. it is a no-op if clustering is not turned off. Someone can infer this by recursively reading the comments on the called functions, but its better to be explicit